### DISCUSSION TITLE
How often is STIG Manager patched for vulnerabilities?

### DISCUSSION THREAD

**Discussion by aport-305** [NUWCDIVNPT/stig-manager#1309] [Q&A]
Hey! I was wanted to see how often STIG Manager is patched and is there anywhere I see the "patch history" to prove its being actively patched? Sorry for the noob question, i'm just trying to prove that its being done to approve the use of the tool in our organization.

**Comment by cd-rite** [comment]
Hi @aport-305  Yep, STIGMan OSS is actively maintained. You can see our Release History here: 
https://github.com/NUWCDIVNPT/stig-manager/releases
and individual commits to the code here:
https://github.com/NUWCDIVNPT/stig-manager/commits/main/
